css/style.css

```diff
@@ -0,0 +1,12 @@
+#img-div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+#image {
+  max-width: 100%;
+  height: auto;
+  text-align: center;
+}
```

Comment:
Please add one space between the CSS classes to enhance the code readability.
Kindly read this [article](https://www.smashingmagazine.com/2008/08/7-principles-of-clean-and-optimized-css-code/) to wrap your head around a clean CSS style sheet. ☺️👀

Reply:
Great job using emojis! :clap: :partying_face: This makes your comments human-friendly :handshake: